src/Debugging/Microsoft.DiaSymReader.PortablePdb/Microsoft.DiaSymReader.PortablePdb.nuspec

```diff
@@ -8,9 +8,9 @@
     </description>
     <dependencies>
       <group targetFramework="portable-net45+win8">
-        <dependency id="Microsoft.DiaSymReader" version="1.0.8-rc3-60429-03" />
-        <dependency id="System.Reflection.Metadata" version="1.2.0-rc3-24102-00" />
-        <dependency id="System.Collections.Immutable" version="1.3.0-rc3-24102-00" />
+        <dependency id="Microsoft.DiaSymReader" version="1.0.7" />
```

Comment:
It doesn't matter. The nuspec is broken in master anyways. I have a change that fixes the versions in master. Waiting for CoreCLR fix to merge it.